.travis.yml

```diff
@@ -10,35 +10,20 @@ cache:
 
 language: ruby
 rvm:
-  - 1.8.7
-  - 1.9.3
```

Comment:
It is broken due to running rails tests (I think, I have checked to make sure). Ideally I'd like to have 1.9.3 non-rails tests run only. I'll deal with it after the refactor though unless you feel like taking care of it.

I am fine getting rid of 1.8.7. As it is I am removing the support in the code during the refactoring.